target/linux/ramips/image/mt7620.mk

```diff
@@ -665,6 +665,16 @@ define Device/microduino_microwrt
 endef
 TARGET_DEVICES += microduino_microwrt
 
+define Device/netcore_nw5212
+  SOC := mt7620a
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Netcore
+  DEVICE_MODEL := NW5212
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci
+  SUPPORTED_DEVICES += nw5212
```

Comment:
this line should probably be dropped, or is there a reason to keep it?

Reply:
I just see this line in context.

Reply:
please don't mark issues as resolved which are not resolved. This looks as if you want to hide shortcomings of your submission, and will bring me into bad mood since I now have to open all of these conversations again and check what else you have to hide.

By the way, how am I supposed to read your answer this way?